x-pack/plugins/security_solution/public/common/hooks/eql/helpers.test.ts

```diff
@@ -604,17 +837,99 @@ describe('eql/helpers', () => {
     });
   });
 
-  describe('getSequenceAggs', () => {
-    test('it aggregates events by sequences', () => {
-      const mockResponse = getMockSequenceResponse();
-      const sequenceAggs = getSequenceAggs(mockResponse, jest.fn() as inputsModel.Refetch);
+  describe('formatInspect', () => {
+    test('it should return "dsl" with response params and index info', () => {
+      const { dsl } = formatInspect(getMockResponse(), ['foo-*']);
+
+      expect(JSON.parse(dsl[0])).toEqual({
+        body: {
+          filter: {
+            range: {
+              '@timestamp': {
+                format: 'strict_date_optional_time',
+                gte: '2020-10-07T00:46:12.414Z',
+                lte: '2020-10-07T01:46:12.414Z',
+              },
+            },
+          },
+        },
+        index: ['foo-*'],
+        method: 'GET',
+        path: '/_eql/search/',
+      });
+    });
+
+    test('it should return "response"', () => {
+      const mockResponse = getMockResponse();
+      const { response } = formatInspect(mockResponse, ['foo-*']);
+
+      expect(JSON.parse(response[0])).toEqual(mockResponse.rawResponse.body);
+    });
+  });
+
+  describe('getEventsToBucket', () => {
+    test('returns events for non-sequence queries', () => {
+      const events = getEventsToBucket(false, getMockResponse());
 
-      expect(sequenceAggs.data).toEqual([
-        { g: 'Seq. 1', x: '2020-10-04T15:16:54.368707900Z', y: 1 },
-        { g: 'Seq. 1', x: '2020-10-04T15:50:54.368707900Z', y: 1 },
-        { g: 'Seq. 2', x: '2020-10-04T15:06:54.368707900Z', y: 1 },
-        { g: 'Seq. 2', x: '2020-10-04T15:15:54.368707900Z', y: 1 },
+      expect(events).toEqual([
+        { _id: '1', _index: 'index', _source: { '@timestamp': '2020-10-04T15:16:54.368707900Z' } },
+        { _id: '2', _index: 'index', _source: { '@timestamp': '2020-10-04T15:50:54.368707900Z' } },
+        { _id: '3', _index: 'index', _source: { '@timestamp': '2020-10-04T15:06:54.368707900Z' } },
+        { _id: '4', _index: 'index', _source: { '@timestamp': '2020-10-04T15:15:54.368707900Z' } },
       ]);
     });
+
+    test('returns empty array if no hits', () => {
+      const resp = getMockResponse();
+      const mockResponse = {
```

Comment:
```suggestion
       const mockResponse: EqlSearchStrategyResponse<EqlSearchResponse<Source>> = {
```

I would type any object once you use a spread as a spread operator ends up degrading the strength of the type to allow additional properties to be added when they shouldn't be.

Before with it where I can add an invalid type:
<img width="622" alt="Screen Shot 2020-10-19 at 12 44 10 PM" src="https://user-images.githubusercontent.com/1151048/96498282-174f2300-1209-11eb-9c76-225a3fccdda8.png">

After with it where I cannot add an invalid type:
<img width="1563" alt="Screen Shot 2020-10-19 at 12 44 25 PM" src="https://user-images.githubusercontent.com/1151048/96498303-1f0ec780-1209-11eb-97a9-53b6e96756c0.png">